daemon/src/main/scala/io/gearpump/cluster/master/AppManager.scala

```diff
@@ -214,6 +213,12 @@ private[cluster] class AppManager(kvService: ActorRef, launcher: AppMasterLaunch
         case AppMasterNonExist =>
           sender ! AppMasterData(AppMasterNonExist)
       }
+
+    case uploadTasks: UploadTaskJar =>
+      uploadTasks.tasks.foreach(uploadedTasks += _ -> uploadTasks.jar)
+      sender ! UploadedTasks(uploadedTasks)
+    case GetUploadedTasks =>
+      sender ! UploadedTasks(uploadedTasks)
```

Comment:
@huafengw @clockfly 
For really big jars (60M+) I found that it killed the master with OOM. I concluded that we needed to stream data in the multipart form which although much more involved would not endanger the master stability. Perhaps check with @clockfly - I think Sean was working on converting spray to http (akka-streams). This PR could be accepted with a note to convert to streaming IO.